README.md

```diff
@@ -88,9 +88,11 @@ Compatible on Windows, macOS & Linux
 
 ### Requirement 2. Rootful mode on Podman Machine
 
-Make sure your `podman machine` has rootful mode enabled.
+Podman Machine requirements:
+* **Rootful mode enabled**
+* Podman Machine must have *at least* 6GB of RAM allocated in order to build the disk image
```

Comment:
"Podman Machine must have " is redundant the line 91, should just start with `* **At least`